indra/tests/test_sources/test_dgi.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+"""Tests for the DGI processor."""
+
+import os
+
+import pandas as pd
+
+from indra.sources.dgi.api import USECOLS
+from indra.sources.dgi.processor import DGIProcessor
+from indra.statements import Inhibition
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+TEST_FILE = os.path.join(HERE, 'resources', 'dgi_sample_interactions.tsv')
```

Comment:
I think this test file needs to be committed.

Reply:
I didn't realize TSV files were ignored by default. Done in 4038d2667